samples/notebooks/quickstart.ipynb

```diff
@@ -0,0 +1,572 @@
+    "      arguments=['--bucket', bucket]\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.3 Create your workflow as a Python function\n",
+    "Start by creating a folder to store the pipeline file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create folders if they don't exist.\n",
+    "!mkdir -p tmp/pipelines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Define your pipeline as a Python function. ` @kfp.dsl.pipeline` is a required decoration including `name` and `description` properties. Then compile the pipeline function. After the compilation is completed, a pipeline file is created."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datetime\n",
+    "import kfp.compiler as compiler\n",
+    "\n",
+    "# Define the pipeline\n",
+    "@kfp.dsl.pipeline(\n",
+    "  name='List GCS Blobs',\n",
+    "  description='Takes a GCS bucket name as input and lists the blobs.'\n",
+    ")\n",
+    "def pipeline_func(bucket=kfp.dsl.PipelineParam('bucket', value='Enter your bucket name here.')):\n",
+    "  list_blobs_task = ListGcsBlobsOp('List', bucket)\n",
+    "\n",
+    "# Compile the pipeline to a file.\n",
+    "filename = 'tmp/pipelines/list_blobs{dt:%Y%m%d_%H%M%S}.pipeline.tar.gz'.format(dt=datetime.datetime.now())\n",
+    "compiler.Compiler().compile(pipeline_func, filename)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Go to your Cloud Shell, run `conn.sh`, click on the provided link. In the new tab that opens, click on \"Pipelines Dashboard\" and go to Kubeflow pipelines UI. Upload the created pipeline and run it.\n",
+    "\n",
+    "**Warning:** When the pipeline is run, it pulls the image from the repository to the Kubernetes cluster to create a container. Kubernetes caches pulled images. One solution is to use the image digest instead of the tag in your component dsl, for example, `s/v1/sha256:9509182e27dcba6d6903fccf444dc6188709cc094a018d5dd4211573597485c9/g`. Alternatively, if you don't want to update the digest every time, you can try `:latest` tag, which will force the k8s to always pull the latest image.."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "___"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 2\n",
+    "# Create a pipeline using Kubeflow Pipelines"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, you will build another component. Then you will see how to connect components to build a multi-component pipeline. You will build the new component by building a Docker container image and wrapping it using `ContainerOp`."
```

Comment:
I'm wondering if we can skip the section above (the 1-component pipeline) and just list this one. It's more useful to build multi-component pipelines anyway

Reply:
This is a quickstart. I think having two parts make it very easy to follow and there is value in having both parts.